.github/workflows/turbo.yml

```diff
@@ -60,14 +60,7 @@ jobs:
 
             - name: Publish highlight npm packages
               if: github.ref == 'refs/heads/main'
-              id: changesets-publish
-              uses: changesets/action@v1
-              with:
-                  publish: yarn publish:highlight
-              env:
-                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-                  NPM_TOKEN: ${{ secrets.NPM_TOKEN_HIGHLIGHT_RUN }}
-                  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN_HIGHLIGHT_RUN }}
+              run: yarn publish:highlight
```

Comment:
The registry is configured above, so I don't think this needs any tokens.

Reply:
🙏  sounds good. one way to test this is to comment out the `if` temporarily that gates this to `refs/heads/main` (pushing a tmp commit that we revert before we merge the PR) so we could run this in the branch